src/Files/Views/Pages/Properties.xaml

```diff
@@ -38,6 +38,36 @@
             VerticalAlignment="Top"
             Background="Transparent" />
 
+        <Grid
+            x:Name="AppWindowTitleBarCaptionButtonsGrid"
+            Grid.Row="0"
+            Height="32"
+            HorizontalAlignment="Right"
+            VerticalAlignment="Top"
+            Background="Transparent">
+            <Rectangle
+                x:Name="CloseRect"
+                Width="46"
+                VerticalAlignment="Stretch"
+                Fill="Transparent"
+                PointerEntered="CloseRect_PointerEntered"
+                PointerExited="CloseRect_PointerExited"
+                PointerPressed="CloseRect_PointerPressed"
```

Comment:
Can you add an automation name for screenreaders?

Reply:
Sure, do we have a localization for the description of a close button?
Or we should just use "Close" (localized ofc)

Reply:
You can reuse the localization that you used for the tooltip.

Reply:
Done